yt_dlp/extractor/mildom.py

```diff
@@ -216,9 +146,8 @@ class MildomVodIE(MildomBaseIE):
     }]
 
     def _real_extract(self, url):
-        m = self._match_valid_url(url)
-        user_id, video_id = m.group('user_id'), m.group('id')
-        url = 'https://www.mildom.com/playback/%s/%s' % (user_id, video_id)
+        user_id, video_id = self._match_valid_url(url).group('user_id', 'id')
+        url = f'https://www.mildom.com/playback/{user_id}/{video_id}'
```

Comment:
why make this a var?
Don't see it being used anywhere other than webpage call...

Reply:
didn't notice until you wrote about it here